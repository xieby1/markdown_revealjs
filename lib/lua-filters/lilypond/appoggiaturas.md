---
lilypond:
  relativize: yes
---

Short appoggiaturas are by no means impracticable in pizzicato. The following
passage from the Scherzo of Beethoven's C-minor Symphony is always executed very
well.

```{.lilypond .ly-fragment ly-name="beethoven" ly-caption="A pizzicato passage from Beethoven's ninth"}
\layout {
    ragged-right = ##t
}

{
    \key c \minor
    \time 3/4
    \tempo "Allegro."
    \relative c'' {
        r4 ees4^"pizz." f4
        f4 f4 g4
        \slashedGrace f8 ees4 d4 c4
        b4 c4 d4
        c4 ees4 f4
        f4 f4 g4
        \slashedGrace f8 ees4 d4 c4 \break
        f4 g4 aes4
        \slashedGrace c,8 b4 a4 g4
        g4 a4 b4
        c4 d4 ees4
        f4 g4 aes4
        aes4 bes4 c4
        c4 c4 b4
        c4 r4 r4 \bar "|."
    }
}
```

Some of our young violinists have learned from Paganini to play rapid descending
scales in pizzicato by plucking the strings with the fingers of the left hand,
which rests firmly on the neck of the violin. They sometimes combine pizzicato
notes (always played with the left hand) with bowed tones, even using the
pizzicato as an accompaniment of a melody played by the bow. All players will
doubtless become familiar with these various techniques in the course of time.
Then composers will be able to take full advantage of them.
