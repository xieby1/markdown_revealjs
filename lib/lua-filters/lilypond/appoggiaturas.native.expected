[ Para
    [ Str "Short"
    , Space
    , Str "appoggiaturas"
    , Space
    , Str "are"
    , Space
    , Str "by"
    , Space
    , Str "no"
    , Space
    , Str "means"
    , Space
    , Str "impracticable"
    , Space
    , Str "in"
    , Space
    , Str "pizzicato."
    , Space
    , Str "The"
    , Space
    , Str "following"
    , SoftBreak
    , Str "passage"
    , Space
    , Str "from"
    , Space
    , Str "the"
    , Space
    , Str "Scherzo"
    , Space
    , Str "of"
    , Space
    , Str "Beethoven\8217s"
    , Space
    , Str "C-minor"
    , Space
    , Str "Symphony"
    , Space
    , Str "is"
    , Space
    , Str "always"
    , Space
    , Str "executed"
    , Space
    , Str "very"
    , SoftBreak
    , Str "well."
    ]
, Para
    [ Image
        ( "" , [ "lilypond-image-standalone" ] , [] )
        [ Str "A pizzicato passage from Beethoven's ninth" ]
        ( "./beethoven.png"
        , "fig:\\layout { ragged-right = ##t } { \\key c \\minor \\time 3/4 \\tempo \"Allegro.\" \\relative c'' { r4 ees4^\"pizz.\" f4 f4 f4 g4 \\slashedGrace f8 ees4 d4 c4 b4 c4 d4 c4 ees4 f4 f4 f4 g4 \\slashedGrace f8 ees4 d4 c4 \\break f4 g4 aes4 \\slashedGrace c,8 b4 a4 g4 g4 a4 b4 c4 d4 ees4 f4 g4 aes4 aes4 bes4 c4 c4 c4 b4 c4 r4 r4 \\bar \"|.\" } }"
        )
    ]
, Para
    [ Str "Some"
    , Space
    , Str "of"
    , Space
    , Str "our"
    , Space
    , Str "young"
    , Space
    , Str "violinists"
    , Space
    , Str "have"
    , Space
    , Str "learned"
    , Space
    , Str "from"
    , Space
    , Str "Paganini"
    , Space
    , Str "to"
    , Space
    , Str "play"
    , Space
    , Str "rapid"
    , Space
    , Str "descending"
    , SoftBreak
    , Str "scales"
    , Space
    , Str "in"
    , Space
    , Str "pizzicato"
    , Space
    , Str "by"
    , Space
    , Str "plucking"
    , Space
    , Str "the"
    , Space
    , Str "strings"
    , Space
    , Str "with"
    , Space
    , Str "the"
    , Space
    , Str "fingers"
    , Space
    , Str "of"
    , Space
    , Str "the"
    , Space
    , Str "left"
    , Space
    , Str "hand,"
    , SoftBreak
    , Str "which"
    , Space
    , Str "rests"
    , Space
    , Str "firmly"
    , Space
    , Str "on"
    , Space
    , Str "the"
    , Space
    , Str "neck"
    , Space
    , Str "of"
    , Space
    , Str "the"
    , Space
    , Str "violin."
    , Space
    , Str "They"
    , Space
    , Str "sometimes"
    , Space
    , Str "combine"
    , Space
    , Str "pizzicato"
    , SoftBreak
    , Str "notes"
    , Space
    , Str "(always"
    , Space
    , Str "played"
    , Space
    , Str "with"
    , Space
    , Str "the"
    , Space
    , Str "left"
    , Space
    , Str "hand)"
    , Space
    , Str "with"
    , Space
    , Str "bowed"
    , Space
    , Str "tones,"
    , Space
    , Str "even"
    , Space
    , Str "using"
    , Space
    , Str "the"
    , SoftBreak
    , Str "pizzicato"
    , Space
    , Str "as"
    , Space
    , Str "an"
    , Space
    , Str "accompaniment"
    , Space
    , Str "of"
    , Space
    , Str "a"
    , Space
    , Str "melody"
    , Space
    , Str "played"
    , Space
    , Str "by"
    , Space
    , Str "the"
    , Space
    , Str "bow."
    , Space
    , Str "All"
    , Space
    , Str "players"
    , Space
    , Str "will"
    , SoftBreak
    , Str "doubtless"
    , Space
    , Str "become"
    , Space
    , Str "familiar"
    , Space
    , Str "with"
    , Space
    , Str "these"
    , Space
    , Str "various"
    , Space
    , Str "techniques"
    , Space
    , Str "in"
    , Space
    , Str "the"
    , Space
    , Str "course"
    , Space
    , Str "of"
    , Space
    , Str "time."
    , SoftBreak
    , Str "Then"
    , Space
    , Str "composers"
    , Space
    , Str "will"
    , Space
    , Str "be"
    , Space
    , Str "able"
    , Space
    , Str "to"
    , Space
    , Str "take"
    , Space
    , Str "full"
    , Space
    , Str "advantage"
    , Space
    , Str "of"
    , Space
    , Str "them."
    ]
]
